examples/create-react-app-material-ui/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "create-react-app-material-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "material-ui": "^1.0.0-alpha.22",
```

Comment:
Regarding next.js as simple `with-next.js` folder and a README pointing toward their repository should be enough.